drop-transfer/src/ws/events.rs

```diff
@@ -177,7 +177,7 @@ impl FileEventTx<IncomingTransfer> {
     }
 
     pub async fn success(&self, final_path: impl Into<PathBuf>) {
-        self.stop(
+        self.force_stop(
```

Comment:
Can you explain this change? I've also taken a look into `stop` and `force_stop` and they seem to be identical? It's difficult to understand what this commit has to do with the commit message to be honest

Reply:
`force_stop()` does not require the file to be started first during the same connection. Look at the `success()` call for the `OutgoingTransfer`.
`stop()` silently ignores the event if the file was not started, it's meant to be used with less critical events like `Progress` for example.

Reply:
My bad, didn't notice that we early return in the `stop` case.